CompanionAppService/Profile.cs

```diff
@@ -29,5 +29,17 @@ public class Profile
 
         /// <summary>Whether this profile describes a currently living commander</summary>
         public bool alive { get; set; }
+
+        /// <summary>The contexts (i.e. "capabilities") associated with this profile </summary>
+        public ProfileContexts contexts { get; set; }
+    }
+
+    public class ProfileContexts
```

Comment:
I am probably missing some context here: what's the benefit of a separate class instead of adding 2 more props to the Profile class?

Reply:
Firstly, because this more closely follows the structure of the profile received from FDev. Secondly, because it makes sense to me to group context information together. Thirdly, because I expect that additional contexts will be added in later updates (the Odyssey expansion, for example, will likely add a new property here).

Reply:
Accepted for the second point.
As for the other 2 points, I believe we should have our own internal model, and map from the FDev model accordingly.